### PR TITLE
DATAREDIS-1158 - Rename unit tests class

### PR DESCRIPTION
**Pull request by solimant**
Rename `RedisElastiCacheConfigurationUnitTests` to `RedisStaticMasterReplicaConfigurationUnitTests` to reflect subject class name; i.e. `RedisStaticMasterReplicaConfiguration`.

- [X] You have read the [Spring Data contribution guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc).
- [X] There is a ticket in the bug tracker for the project in our [JIRA](https://jira.spring.io/browse/DATAREDIS).
- [X] You use the code formatters provided [here](https://github.com/spring-projects/spring-data-build/tree/master/etc/ide) and have them applied to your changes. Don’t submit any formatting related changes.
- [X] You submit test cases (unit or integration tests) that back your changes.
- [X] You added yourself as author in the headers of the classes you touched. Amend the date range in the Apache license header if needed. For new types, add the license header (copy from another file and set the current year only).

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@solimant Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-data-redis&pullRequestId=535)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-data-redis&pullRequestId=535) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@solimant Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-data-redis&pullRequestId=535)!

File: src/test/java/org/springframework/data/redis/connection/RedisElastiCacheConfigurationUnitTests.java [hunk 1]
```diff
@@ -23,8 +23,9 @@
  * Unit tests for {@link RedisStaticMasterReplicaConfiguration}.
  *
  * @author Mark Paluch
+ * @author Tamer Soliman
```
Comment (solimant) [inline src/test/java/org/springframework/data/redis/connection/RedisElastiCacheConfigurationUnitTests.java:26]: @mp911de - I don't consider myself _worthy_ of adding my name as an "author" for such an utterly trivial change, but I'm only strictly following the [guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc#quickstart):
> Make sure you added yourself as author in the headers of the classes you touched.

Wouldn't be hurt if you removed it 🙂

**Comment by mp911de** [comment]
The test has intentionally been named this way to indicate what it's for. Therefore, we do want to keep things as-is.

**Pull request closed without merging**